generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -187,8 +179,7 @@ Scenario: Running a 'containingRegex' request that includes only anchor ^ should
        | "c"  |
 
 Scenario: Running a 'containingRegex' request that includes only anchor $ should be successful
-     Given there is a field foo
-       And foo is containing regex /[a-c]{1}$/
+     Given foo is containing regex /[a-c]{1}$/
```

Comment:
do we need to update the scenario descriptions for these tests ?

Reply:
@mrspaceman would the description for the scenario be more meaningful to be written as:
containingRegex that includes the closing anchor should be successful?

Reply:
maybe `that does not include the opening anchor ^` ?

Reply:
Updated in line with suggestion @mrspaceman